settlement/bitflyer/upload.go

```diff
@@ -237,38 +236,36 @@ func setupSettlementTransactions(
 			}
 		}
 		if !aggregatedTx.Probi.Equals(decimal.Zero) {
+			aggregatedTx.Probi = altcurrency.BAT.ToProbi(altcurrency.BAT.FromProbi(aggregatedTx.Probi).Truncate(8))
```

Comment:
let's add a quick comment here that bitflyer wants us to round to nearest `0.00000001`, otherwise we may run into API errors

Reply:
done